stories/constants/propTypes.ts

```diff
@@ -25,3 +26,11 @@ export interface AvatarProps {
 
   image: IMAGE;
 }
+
+export type IconMenuProps ={
+  hastext?: boolean;
+  label: string;
+  icon: ICONMENU;
```

Comment:
```suggestion
  icon: ICON_MENU;
```
Nếu là khai báo constant thì viết hoa hết nên dùng dấu _ để phân cách các chữ